modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/ManualGroupUpdateRequest.java

```diff
@@ -359,7 +359,8 @@ private static void enrichAssignments(
             int replicas,
             Set<Assignment> partAssignments
     ) {
-        Set<Assignment> calcAssignments = AffinityUtils.calculateAssignmentForPartition(aliveDataNodes, partId.partitionId(), replicas);
+        Set<Assignment> calcAssignments = PartitionDistributionUtils.calculateAssignmentForPartition(aliveDataNodes, partId.partitionId(),
```

Comment:
Here and there. I'd rather use static imports in such cases - calculateAssignmentForPartition name is self-evident.

Reply:
done